lib/classes/class-settings.php

```diff
@@ -127,6 +127,48 @@ public function __construct($args = false) {
           }
           $this->set('property_types', $ar);
         }
+        
+        /* Standard Attributes 
+         * added 28/07/2016 @raj
+         * 
+         */
+
+        $PDF_unique = array(
```

Comment:
@raj-uimatic I believe this is the list of standard attributes? If so, can we have this configuration come from a JSON file? We need to make it easily updatable.